### ISSUE TITLE
LOW_PRIORITY: Firefox doesn't recognize new magnet links

### ISSUE THREAD

**Issue by Kreyren** [bedrocklinux/bedrocklinux-userland#111]
## Current
Installed playonlinux on debian buster stratum assuming firefox on gentoo stratum -> Firefox is unable to open `playonlinux://www.playonlinux.com/repository/download_update.php?id=9880`.

## Expected
if hyperlink that generates `playonlinux://...` is invoked -> open said program.

## HotFix
Based on https://support.mozilla.org/en-US/questions/1255394?utm_campaign=questions-reply&utm_medium=email&utm_source=notification Manually updating `~/.confing/.mozilla/firefox/<USER_ID>/handlers.json` is required.

## Fix
Unknown

**Comment by paradigm** [comment]
Have you tried lessening the sandbox strictness?

https://bedrocklinux.org/0.7/compatibility-and-workarounds.html#firefox-fonts

**Comment by paradigm** [comment]
Reading around, I see lots of reports that playonlinux does not register as a protocol with various distros.  For example, here: https://bugs.archlinux.org/task/51810

Do you know if the Debian buster playonlinux package registers its protocol handler with Firefox on a normal Debian Buster install, without Bedrock involved?

**Comment by Kreyren** [comment]
@paradigm I've applied workaround, but that was unique scenario in hotfix. 

This issue seems to be affecting most of magnet links i had this issue with torrent magnet links, but that doesn't seem to be present anymore assuming firefox-esr from debian buster and  qbittorent from gentoo.

I've noticed 
```
(/usr/lib/firefox-esr/firefox-esr:20051): dconf-WARNING **: 06:44:52.132: Unable to open /bedrock/cross/dconf/profile/user: Permission denied
```

In terminal not sure if it's relevant so consider this as low priority and i will report back to you with more info.
